nvme-print-stdout.c

```diff
@@ -3264,20 +3278,21 @@ static void stdout_zns_changed(struct nvme_zns_changed_zone_log *log)
 
 static void stdout_zns_report_zone_attributes(__u8 za, __u8 zai)
 {
-	const char *const recommended_limit[4] = {"","1","2","3"};
+	const char *const recommended_limit[4] = {"", "1", "2", "3"};
```

Comment:
Add a space between `*` and `const` while you're touching this line? `const char * const`

Reply:
Yes done it.